File: Perf.java
public class Perf
{
    private static final int WARMUP = 5;
    private static final int SECONDS = 10;

    public static void main(String[] args)
    {
        int passes = 0;
        long tick = System.currentTimeMillis();

        while (System.currentTimeMillis() - tick < WARMUP * 1000)
        {
            var t = GetPrimesUnderN(10000 - (passes % 10000));
            if (t > 1229) System.out.println("Incorrect number of primes");
        }

        tick = System.currentTimeMillis();

        while (System.currentTimeMillis() - tick < SECONDS * 1000)
        {
            var t = GetPrimesUnderN(10000 - (passes % 10000));
            if (t > 1229) System.out.println("Incorrect number of primes");
            passes++;
        }

        System.out.println("Passed " + passes + " runs in " + (System.currentTimeMillis() - tick) / 1000 + " seconds.");
    }

    private static int GetPrimesUnderN(int n)
    {
        int primes = 0;

        for (int i = 2; i < n; i++)
            if (IsPrime(i))
                primes++;

        return primes;
    }

    private static boolean IsPrime(int i)
    {
        for (int j = 2; j < i; j++)
            if (i % j == 0) 
                return false;

        return true;
    }
}

File: perf.py
import time;

WARMUP = 5
SECONDS = 10

def IsPrime(i):
	for j in range(2, i):
		if i % j == 0:
			return False

	return True

def GetPrimesUnderN(n):
	primes = 0

	for i in range(2, n):
		if IsPrime(i):
			primes += 1

	return primes

def main():
	passes = 0
	tick = time.time()

	while time.time() - tick < WARMUP:
		t = GetPrimesUnderN(10000 - (passes % 10000))
		if t > 1229:
			print("Incorrect number of primes.")

	tick = time.time()

	while time.time() - tick < SECONDS:
		t = GetPrimesUnderN(10000 - (passes % 10000))
		if t > 1229:
			print("Incorrect number of primes.")
		passes += 1

	print("Passed " + str(passes) + " runs in " + str(time.time() - tick) + " seconds.")

main()